### ISSUE TITLE
MSC "add all" option

### ISSUE THREAD

**Issue by ryness** [munki/munki#518]
In Managed Software Center updates available show up on the Updates tab with plus signs next to them ("Other available updates). Each click of the plus sign bounces the user to a screen showing progress info. Once the progress is done the user is bounced back to the Updates screen with the item they clicked the plus-sign for in a separate section above. The Update All button only updates "all" of the apps listed in the section above. This is confusing to the user who thinks that Update All would actually update everything in front of their face. It's also time consuming for the user if there are a large number of updtes and they need to click the plus-sign and wait for each individual item's progress bar before it appears in the upper section. Seems an Add All or similar button, or the ability to select multiple items, allowing users to add all or select optional updates would be pretty handy.
![synca](https://cloud.githubusercontent.com/assets/13770089/10005551/4c8e97dc-6064-11e5-829d-4ec5b1d4435f.jpg)

**Comment by gregneagle** [comment]
This appears to be a duplicate of https://github.com/munki/munki/issues/334
